src/main/java/com/intel/genomicsdb/GenomicsDBImporter.java

```diff
@@ -731,23 +649,42 @@ static File printLoaderJSONFile(
    *
    * Assume one sample per input GVCF file
    *
-   * @param  variants  Variant Readers objects of the input GVCF files
+   * @param  sampleToReaderMap  Variant Readers objects of the input GVCF files
    * @param useSamplesInOrderProvided  If True, do not sort the samples,
    *                                   use the order they appear in
    * @param lbRowIdx Smallest row idx which should be imported by this object
    * @return  Mappings of callset (sample) names to TileDB rows
    */
   static GenomicsDBCallsetsMapProto.CallsetMappingPB generateSortedCallSetMap(
```

Comment:
I'm confused about this method.  It seems like you either should have a pre-validated mapping of sample -> feature reader in which case this is pointless and needlessly expensive, or you should have a list of FeatureReaders and no sample names.  I don't understand the case where you already have a mapping and you want to just run through it and check it again.  That seems needlessly expensive and pointless.  It might make sense to make the method available as a void validateSampleMap that could be optionally called, but I don't think want it in the main constructor.  
Part of the point of providing a sample name -> featurereader map is that you might want to override the sample names in the actual file because they are not unique.  Making this validation mandatory makes that impossible.